pkgs/games/MarbleMarcher/default.nix

```diff
@@ -0,0 +1,97 @@
+{ lib, stdenv, fetchFromGitHub, runtimeShell, cmake, eigen, sfml, libglvnd }:
+
+let
+  pname = "MarbleMarcher";
+
+  # The game doesn’t have any versions in GitHub.
+  # Just taking a commit from the top of the “master” branch.
+  version = "git-2019-04-22";
```

Comment:
```suggestion
  pname = "MarbleMarcher";
  version = "unstable-2019-04-22";
```

Reply:
Why would you remove that comment? This is useful info for the context of versioning this application. I intentionally tried to give a hint for my future self. Or anyone else who is going to update the package.

Reply:
Also I don’t think it makes it be “unstable”. The project just doesn’t have any versioning policy except commits themselves.

Reply:
It is a standard practice that if the package has no stable versioning that we take the latest commit from the default branch.

> Also I don’t think it makes it be “unstable”.

It does by our definition. If there are no stable release tags/versions then the package is by our definition unstable.

Reply:
@SuperSandro2000 I renamed version to `unstable-2019-04-22`.